requirements.txt

```diff
@@ -3,6 +3,7 @@ django_readonly_field
 jsonschema>=3.0
 libcomps~=0.1.15
 productmd>=1.25
-pulpcore>=3.7
+pulpcore>=3.10
 PyGObject~=3.22
 solv~=0.7.17
+aiohttp_xmlrpc
```

Comment:
@dralley , @jlsherrill, @evgeni , JFYI, when this is released (likely pulp_rpm 3.10), it will bump the pulpcore version requirement to 3.10, and also it adds a new dependency (in case @evgeni will need to build it for rpm installations).

Reply:
I will have to, but thankfully we have tools to catch that.

Thanks for the ping tho :)